common/thrift_client_pool.h

```diff
@@ -40,6 +40,8 @@ DECLARE_int32(channel_max_checking_size);
 
 DECLARE_int32(channel_send_timeout_ms);
 
+DECLARE_int32(thrift_client_pool_threads);
```

Comment:
can we rename it to default_thrift_client_pool_threads?

Because client can pass in a different value to ThriftClientPool() to overwrite the value provided by this flag.